pdp-server/src/config/mod.rs

```diff
@@ -27,8 +27,8 @@ pub struct PDPConfig {
     #[serde(default)]
     pub use_new_authorized_users: bool,
 
-    /// Timeout in seconds for health checks (default: 1.0 second)
-    #[serde(default)]
+    /// Timeout in seconds for health checks (default: 3 second)
+    #[serde(default = "default_healthcheck_timeout")]
```

Comment:
Why do you have it here ? You don't have the default set like this for any other property